HISTORY.md

```diff
@@ -1,5 +1,11 @@
 # History
 
+## 2.2.0
```

Comment:
This doesn't actually need to be done as part of the PR, and is done when releasing it